[2022-12-27 16:40:51,317] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: crypto_data_etl_auto_train_model.get_crypto_trading_data 2022-12-26T07:00:00+00:00 [queued]>
[2022-12-27 16:40:51,332] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: crypto_data_etl_auto_train_model.get_crypto_trading_data 2022-12-26T07:00:00+00:00 [queued]>
[2022-12-27 16:40:51,332] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-12-27 16:40:51,333] {taskinstance.py:1043} INFO - Starting attempt 2 of 7
[2022-12-27 16:40:51,333] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-12-27 16:40:51,345] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): get_crypto_trading_data> on 2022-12-26T07:00:00+00:00
[2022-12-27 16:40:51,353] {standard_task_runner.py:52} INFO - Started process 31162 to run task
[2022-12-27 16:40:51,359] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'crypto_data_etl_auto_train_model', 'get_crypto_trading_data', '2022-12-26T07:00:00+00:00', '--job-id', '1087', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl.py', '--cfg-path', '/tmp/tmp7w9pvmh5', '--error-file', '/tmp/tmpvtxhdsto']
[2022-12-27 16:40:51,364] {standard_task_runner.py:77} INFO - Job 1087: Subtask get_crypto_trading_data
[2022-12-27 16:40:51,410] {logging_mixin.py:104} INFO - Running <TaskInstance: crypto_data_etl_auto_train_model.get_crypto_trading_data 2022-12-26T07:00:00+00:00 [running]> on host 2a76951a993b
[2022-12-27 16:40:51,446] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=VMD
AIRFLOW_CTX_DAG_ID=crypto_data_etl_auto_train_model
AIRFLOW_CTX_TASK_ID=get_crypto_trading_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-26T07:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-26T07:00:00+00:00
[2022-12-27 16:40:54,438] {logging_mixin.py:104} INFO - Error: duplicate key value violates unique constraint "sentiment_pkey"
DETAIL:  Key (date)=(2022-12-26) already exists.
[2022-12-27 16:40:54,439] {logging_mixin.py:104} INFO - 
[2022-12-27 16:40:54,443] {logging_mixin.py:104} INFO - Error: duplicate key value violates unique constraint "price_and_volumn_pkey"
DETAIL:  Key (date)=(2022-12-26) already exists.
[2022-12-27 16:40:54,443] {logging_mixin.py:104} INFO - 
[2022-12-27 16:40:54,444] {python.py:118} INFO - Done. Returned value was: None
[2022-12-27 16:40:54,451] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=crypto_data_etl_auto_train_model, task_id=get_crypto_trading_data, execution_date=20221226T070000, start_date=20221227T164051, end_date=20221227T164054
[2022-12-27 16:40:54,468] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-27 16:40:54,506] {local_task_job.py:146} INFO - Task exited with return code 0
